web-animations-1/Overview.bs

```diff
@@ -721,16 +730,30 @@ time</dfn>.
 Issue(2079): There must be a better term than &ldquo;origin time&rdquo;&mdash;
              it's too similar to &ldquo;time origin&rdquo;.
 
-Prior to establishing the [=time origin=] for its associated document,
-a [=document timeline=] is <a lt="inactive timeline">inactive</a>.
+
+The [=timeline phase|phase=] of a <a>document timeline</a> at a given moment is
+the [=timeline phase|phase=] corresponding to the <em>first</em> matching
+condition from the following:
+
+<div class="switch">
+
+:   If the associated document is not an [=active document=] or its
+    [=time origin=] has not yet been established
+::  &rarr; [=timeline inactive phase|inactive=]
+:   Otherwise,
+::  &rarr; [=timeline active phase|active=]
+
+</div>
+
+A [=document timeline=] in the [=timeline inactive phase|inactive phase=] can
+also be called an <dfn>inactive timeline</dfn>
```

Comment:
Is this definition supposed to be limited to document timelines? I think it applies to all types of timelines, right?

Reply:
Great catch, moving this up to the timeline phase section.